### PR TITLE
Update faq.md

### PR DESCRIPTION
**Pull request by java-spaces**
:clipboard: Add associated issues, tickets, docs URL here.

## Overview

I've added the completed Dev FAQ and added it here as per what Ino and I discussed.

## Changes

- I've added the entire Dev FAQ to the bottom of the page, including images and links to the documentation wherever necessary so that developers can better understand how to make use of everything available to them in the Boba Network

### CODE REVIEW

File: faq.md [hunk 1]
```diff
@@ -105,3 +105,195 @@ There are 4 different mechanisms for following the status of a transaction.
...
+### Q3: How Can I Pay for Fees with BOBA Via an API?
+Below is the js code needed to utilize the API:
+
+```
+const registerTx = await Boba_GasPriceOracle.useBobaAsFeeToken()
+   await registerTx.wait()
+expect(
+     await Boba_GasPriceOracle.bobaFeeTokenUsers(env.l2Wallet.address)
+   ).to.be.deep.eq(true)
+
```
Comment (boyuan-chen) [inline faq.md]: This is the test code. We don't need to include this.

File: faq.md [hunk 2]
```diff
@@ -105,3 +105,195 @@ There are 4 different mechanisms for following the status of a transaction.
...
+Once you’re finished, hit “Verify & Publish”
+
+
+<kdb><img width="598" alt="Screen Shot 2022-08-16 at 3 35 35 PM" src="https://user-images.githubusercontent.com/107710263/184893126-3f0197fb-caf5-46d6-9c90-1aa091cbc245.png"></kbd>
+
+
+Alternatively, you could also [use services like Souricify](https://sourcify.dev/) to verify your smart contract as well.
+
+The full details of this process [can be found in the docs](https://docs.boba.network/for-developers/verify-smart-contracts)
```
Comment (boyuan-chen) [inline faq.md]: This is out of date. We are not using blockscout. Users can use the bobascan to verify the contracts or use the `[hardhat-etherscan](https://hardhat.org/hardhat-runner/plugins/nomiclabs-hardhat-etherscan#hardhat-etherscan)` plug to verify contracts.

**Review by boyuan-chen — CHANGES_REQUESTED** [review]

File: faq.md (hunk 2 above)
Comment (java-spaces) [reply faq.md]: Ah, right. I'll delete this for now until I can get some new screenshots and included it in a v2

File: faq.md (hunk 1 above)
Comment (java-spaces) [reply faq.md]: Thanks for catching that, deleted now.

**Review by InoMurko — APPROVED** [review]

**Review by boyuan-chen — APPROVED** [review]

**Pull request merged**